code/modules/admin/admin.dm

```diff
@@ -1067,3 +1067,28 @@ var/gamma_ship_location = 1 // 0 = station , 1 = space
 			continue
 		result[1]++
 	return result
+
+//Discord duplications
+/datum/admins/proc/discord_duplicates()
+	if(!usr.client.holder)
+		return
+	var/dat = "<html><head><title>Discord Duplicates</title></head>"
+	dat += "<body><p><i>Discord IDs with more than one ckey linked are shown below</i></i><table border=1 cellspacing=5><B><tr><th>Discord ID</th><th>CKEYs</th><th>Unlink</th></B>"
+	// If anyone reads this, I spent a whole 30 minutes writing just this fucking query. It is the messiest SQL statement I have ever written
+	// If anyone even thinks about touching this I will impale you on a railroad spike

```

Comment:
Railroad spikes are too small to impale people with, this is an oversight. Please amend.